.circleci/config.yml

```diff
@@ -1035,41 +1043,53 @@ jobs:
       - DEPLOY_DOWNTIME: 1800 #30M
     steps: *deploy_steps
 
+  # Upload artifacts
   upload_packages_osx:
-    executor: infrastructure_container_stable
```

Comment:
I prefer to keep the explicit executors for each job, with CircleCI 2.1 it's much readable than current mix of partials